js/CRM_Contact_Page_View_UserDashBoard.js

```diff
@@ -17,5 +17,33 @@
     var html = $(el).html().replace(/[\[\]]/g , '');
     $(el).html(html);
   })
+
+  // Change renew button link to match the users current membership type
+  $(window).load(function(){
+    // Get the membership type
+    var memberShipType = $('#memberships').find('.form-item table tbody tr:not(.columnheader) td:first-child').html();
```

Comment:
This logic assumes there will only be one Membership Type listed on the page. But what if the user has both a "Canada" membership type and a "US" membership type? Consider the attached image as an example.  ![multiple-types](https://user-images.githubusercontent.com/759449/88702312-c7146900-d0d0-11ea-9980-fb0e972fed86.png)
To test this, you could open that Contact record and follow the 1-2-3-4 steps in the second screenshot (1. click the Memberhips tab; 2 click Add membership; 3. select a US membership type; 4. click save) then reload the user's dashboard.
![add-membership](https://user-images.githubusercontent.com/759449/88702362-d5fb1b80-d0d0-11ea-876d-6e722fa0f6a5.png)

Reply:
Done